src/externalized/mercs/MercProfile.cc

```diff
@@ -178,36 +207,52 @@ std::unique_ptr<MERCPROFILESTRUCT> MercProfile::deserializeStruct(const JsonValu
 		prof->bSexist = SexistLevels::VERY_SEXIST;
 	}
 
+	JsonObject stats;
 	if (r.has("stats")) {
-		JsonObject stats = r["stats"].toObject();
-		ST::string jEvolution = stats.getOptionalString("evolution");
-		if (jEvolution == "NONE") {
-			prof->bEvolution = CharacterEvolution::NO_EVOLUTION;
-		}
-		else if (jEvolution == "REVERSED") {
-			prof->bEvolution = CharacterEvolution::DEVOLVE;
-		}
-		prof->bLifeMax = stats.getOptionalUInt("health", 15);
-		prof->bLife = stats.getOptionalUInt("health", 15);
-		prof->bAgility = stats.getOptionalUInt("agility", 1);
-		prof->bDexterity = stats.getOptionalUInt("dexterity", 1);
-		prof->bStrength = stats.getOptionalUInt("strength", 1);
-		prof->bLeadership = stats.getOptionalUInt("leadership", 1);
-		prof->bWisdom = stats.getOptionalUInt("wisdom", 1);
-		prof->bExpLevel = stats.getOptionalUInt("experience", 1);
-		prof->bMarksmanship = stats.getOptionalUInt("marksmanship");
-		prof->bMedical = stats.getOptionalUInt("medical");
-		prof->bExplosive = stats.getOptionalUInt("explosive");
-		prof->bMechanical = stats.getOptionalUInt("mechanical");
-		prof->ubNeedForSleep = stats.getOptionalUInt("sleepiness", 7);
-	}
-	prof->bPersonalityTrait = Internals::getPersonalityTraitEnumFromString(r.getOptionalString("personalityTrait"));
-	prof->bSkillTrait = Internals::getSkillTraitEnumFromString(r.getOptionalString("skillTrait"));
-	prof->bSkillTrait2 = Internals::getSkillTraitEnumFromString(r.getOptionalString("skillTrait2"));
-	prof->bAttitude = Internals::getAttitudeEnumFromString(r.getOptionalString("attitude"));
+		stats = r["stats"].toObject();
+	}
+	ST::string jEvolution = stats.getOptionalString("evolution");
+	if (jEvolution.empty()) {
+		prof->bEvolution = binaryProf->bEvolution;
+	}
+	else if (jEvolution == "NONE") {
+		prof->bEvolution = CharacterEvolution::NO_EVOLUTION;
+	}
+	else if (jEvolution == "REVERSED") {
+		prof->bEvolution = CharacterEvolution::DEVOLVE;
+	}
```

Comment:
Probably worth at least a comment here that normal evolution does not need to be handled here because *prof was default initialized and NORMAL_EVOLUTION's numerical value is 0.